src/vm/amd64/unixasmhelpers.S

```diff
@@ -304,7 +232,7 @@ NESTED_ENTRY OnHijackInteriorPointerTripThread, _TEXT, NoHandler
 
     END_PROLOGUE
 
-    call                C_FUNC(OnHijackInteriorPointerWorker)
+    call                C_FUNC(OnHijackWorker)
 
     free_stack          0x8
     pop_register        rax
```

Comment:
How does the GC update the RDX if it needs to? The original OnHijackStructInRegsTripThread was pushing RAX and RDX.